### PR TITLE
Added quiz for rom microcontroller

### PR DESCRIPTION
**Pull request by Samridhi-98**
Added quiz for rom-microcontroller.md

Fixes #505 

#### Changes done:
- [x] Task 1
MCQ in `rom-microprocessor.md`

#### Screenshots
csv

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@Shivansh2407 Please review this PR

File: docs/binary-algebra/addition.md [hunk 1]
```diff
@@ -46,3 +46,4 @@ In the above example, for units place gives 1 as the submission of 1 and 0, wher
 1 1 0   (6)
 -----
 ```
+
```
Comment (ayan-biswas0412) [inline docs/binary-algebra/addition.md:49]:
```suggestion
```

File: docs/comb-ssi/universal-gates.md [hunk 2]
```diff
@@ -148,3 +148,6 @@ F = (A NOR (B NOR B)) NOR ((C NOR C) NOR D)
 Notice that there are input elements that are present in the negative form, namely `B'` and `C'`. You can represent them by using the NOR gate in order to realise the [NOT gate](https://learn.circuitverse.org/docs/universal_gates.html#implementing-not-gate).
 
 <iframe width="600px" height="300px" src="https://circuitverse.org/simulator/embed/97663" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen></iframe>
+
```
Comment (ayan-biswas0412) [inline docs/comb-ssi/universal-gates.md:151]: please remove unnecessary spaces

File: docs/logic-design/canonical.md [hunk 3]
```diff
@@ -365,3 +365,5 @@ The following methods can be used to simplify the the Boolean function:
 
 1. The [Karnaugh-map](https://learn.circuitverse.org/docs/maps.html) or K-map method.
 1. The [NAND gate method](https://learn.circuitverse.org/docs/nand_gate_method.html).
+
```
Comment (ayan-biswas0412) [inline docs/logic-design/canonical.md:368]: here also

File: docs/logic-design/functional-description.md [hunk 4]
```diff
@@ -33,3 +33,7 @@ This combinational circuit has ‘n’ input variables and ‘m’ outputs. Each
 1.  Formulate the Truth table. If there are ‘n’ input variables, then there will be 2n possible combinations. For each combination of input, find the output values.   
 1.  Find the Boolean expressions for each output. If necessary, simplify those expressions.   
 1.  Implement the above Boolean expressions corresponding to each output by using Logic gates.
+
```
Comment (ayan-biswas0412) [inline docs/logic-design/functional-description.md:36]: please remove this blank space lines

File: docs/seq-lsi/rom-microcontrollers.md [hunk 5]
```diff
@@ -64,3 +64,23 @@ For FSM which do not exhibit a simple sequence of states, the counter can be rep
 ![MUX based microcontroller](/assets/images/seq/rom_mux.svg)
 
 ROM capacity: $2^N \times (2N+K+\log_2 M)$
+
+{:.quiz}
+
+1. What is the ROM capacity of Counter based µcontroller?
```
Comment (ayan-biswas0412) [inline docs/seq-lsi/rom-microcontrollers.md:70]:
```suggestion
1. What is the ROM capacity of Counter based µcontroller ?
```
in each of the question space before the question mark must be added

File: docs/seq-lsi/rom-microcontrollers.md [hunk 6]
```diff
@@ -64,3 +64,23 @@ For FSM which do not exhibit a simple sequence of states, the counter can be rep
...
+   1. 2^N\times(K+\log_2 M)
+   * 2^N\times(N+K+\log_2 M)
+   * 2^{(M+N)}\times(K+N)
+   * 2^M\times(M+N)
+
+2. Which among the following is not micro-controller?
```
Comment (ayan-biswas0412) [inline docs/seq-lsi/rom-microcontrollers.md:76]:
```suggestion
2. Which among the following is not micro-controller ?
```

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
1.please remove the unnecessary spaces
2. add single space before question mark in each question

File: docs/seq-lsi/rom-microcontrollers.md (hunk 6 above)
Comment (Samridhi-98) [reply docs/seq-lsi/rom-microcontrollers.md]: @ayan-biswas0412 Made all the changes

**Comment by Samridhi-98** [comment]
@Shivansh2407  Will you review my PR.

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
please resolve the conflicts

**Comment by Samridhi-98** [comment]
@ayan-biswas0412  @Shivansh2407  conflict has been resolved

**Review by Shivansh2407 — APPROVED** [review]
Great work @Samridhi-98 🎉 I would suggest to squash the commits before you finalize the PR for review in future. Thank you!

**Pull request merged**